packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistory.tsx

```diff
@@ -77,7 +77,7 @@ export const TransactionHistory = ({
               {/* Deposits */}
               <img
                 src={getNetworkLogo(l2.network.chainID)}
-                className="max-w-6 h-6"
+                className="h-6 max-w-6"
```

Comment:
All the class name changes are actually rearrangements done by `prettier:format` once custom width was added in talwind-config.